### PR TITLE
Expose Workflow Checkpoint in Workflow Reports

### PR DESCRIPTION
**Pull request by assuntad23**
Part of #16556 
Exposing the workflow version as workflow checkpoint on report. See comments in #16682 for reasoning. 

## How to test the changes?
(Select all options that apply)
- [X] Instructions for manual testing are as follows:
  1. Create a Workflow Report
  2. Note how Workflow Checkpoint is below the Title
  3. Note: if you do not see the Checkpoint, that is because it is waiting on the store to provide it, refreshing may work.  But I have it rendering conditionally, so if it doesn't return from the store in time, nothing is printed there instead. 

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

**Comment by nsoranzo** [comment]
@davelopez suggested to call them "revisions" instead of checkpoints, as we do in pages and toolshed, was that ruled out?

File: client/src/components/Markdown/Markdown.vue [hunk 1]
```diff
@@ -27,6 +27,9 @@
                     <h1 class="text-break align-middle">
                         Title: {{ markdownConfig.title || markdownConfig.model_class }}
                     </h1>
+                    <h3 v-if="workflowVersions" class="text-break align-middle">
+                        Workflow Checkpoint: {{ workflowVersions.version }}
+                    </h3>
```
Comment (ElectronicBlueberry) [inline client/src/components/Markdown/Markdown.vue]:
```suggestion
                    <h2 v-if="workflowVersions" class="text-break align-middle">
                        Workflow Checkpoint: {{ workflowVersions.version }}
                    </h2>
```

[See here for more details](https://www.w3.org/WAI/tutorials/page-structure/headings/)

File: client/src/components/Markdown/Markdown.vue [hunk 2]
```diff
@@ -162,8 +172,10 @@ export default {
     },
     created() {
         this.initConfig();
+        this.getWFversion();
```
Comment (ElectronicBlueberry) [inline client/src/components/Markdown/Markdown.vue]:
```suggestion
        this.fetchWorkflowForInstanceId(this.workflowID);
```

**Review by dannon — CHANGES_REQUESTED** [review]
~~This looks like it likely breaks basic pages?  They won't have a workflow version or any of this stuff.~~

~~(actually, going to test this -- are pages also just broken in dev as-is?)~~

Okay, scratch that -- what breaks basic pages displaying is creating a new page with zero content and going to view it.  We need to handle that part better, but it was unrelated to the changes here -- I'll follow up separately with a fix for that.

**Review by dannon — APPROVED** [review]

**Pull request merged**